src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -4,12 +4,16 @@ import classNames from 'classnames';
 
 import IconSearch from '@department-of-veterans-affairs/formation/IconSearch';
 import DropDownPanel from '@department-of-veterans-affairs/formation/DropDownPanel';
+import isBrandConsolidationEnabled from '../../../brand-consolidation/feature-flag';
+
 
 class SearchMenu extends React.Component {
   constructor(props) {
     super(props);
     this.makeForm = this.makeForm.bind(this);
     this.toggleSearchForm = this.toggleSearchForm.bind(this);
+    this.searchAction = isBrandConsolidationEnabled() ? 'https://search.usa.gov/search' : 'https://search.vets.gov/search';
+    this.searchAffiliate = isBrandConsolidationEnabled() ? 'va' : 'vets.gov_search';
```

Comment:
This will work but I think it's more idiomatic to put data like this into `this.state` in the constructor rather than on the React component instance itself since it's used as part of the render update.

Reply:
Thanks @rianfowler  -- something like this?

```jsx
class SearchMenu extends React.Component {
  constructor(props) {
    super(props);
    this.makeForm = this.makeForm.bind(this);
    this.toggleSearchForm = this.toggleSearchForm.bind(this);
    this.state = {
      searchAction: isBrandConsolidationEnabled() ? 'https://search.usa.gov/search' : 'https://search.vets.gov/search',
      searchAffiliate: isBrandConsolidationEnabled() ? 'va' : 'vets.gov_search'
    };
  }

// ...

      <form
        acceptCharset="UTF-8"
        action={this.state.searchAction}
        id="search"
        method="get">
```

Reply:
Exactly.